### ISSUE TITLE
Override default error for authentication

### ISSUE THREAD

**Issue by omidraha** [vitalik/django-ninja#482]
I know about custom exception in [authentication](https://django-ninja.rest-framework.com/tutorial/authentication/#custom-exceptions) and error [handling](https://django-ninja.rest-framework.com/tutorial/errors/) ,

But is it possible to override this [line](https://github.com/vitalik/django-ninja/blob/f58dfc685189b04f2849eece43171e91eafaa3b2/ninja/operation.py#L152) of code:
```python
return self.api.create_response(request, {"detail": "Unauthorized"}, status=401)
```
Currently I use `HttpBearer` class and I have `api.exception_handler`, 
And for wrong token I get my custom error message like this:
```python
{
  "message": "Invalid token"
}
```

But still for missing token I get this error:

```python
{
  "detail": "Unauthorized"
}
```

**Comment by SmileyChris** [comment]
This was fixed in #454

**Comment by omidraha** [comment]
Thanks.
